src/routes/index.svelte.md

```diff
@@ -43,6 +43,10 @@ https://web.archive.org/web/20150409040851/https://manjaro.github.io/expired_SSL
 
 Holding back packages for two weeks can also cause security issues, but this issue is probably better addressed in the stability section.
 
+## "Suitable for beginners"
+
+Manjaro claims that it is a distribution that's suitable for both beginners and experienced users, this is also an opinion you can often hear on the internet. In reality, beginners shouldn't use rolling-release distros. If you're using one you need to be prepared for potential issues and bugs and know how to troubleshoot them because of the very frequent updates - with rolling-release distros you get the latest versions of packages which, while it might sound good, may result in problems with the packages. Frequent kernel updates can sometimes result in issues with drivers, and you have to troubleshoot these issues as well.
```

Comment:
i'm skeptical of the idea beginners shouldn't use rolling distros -- rolling just means there are no set release candidates with an EoL. bugs are also possible with upgrading packages in leap style distros. its more important how well tested everything is and how close to the edge a distro rides -- thats more on the distro's individual protocol rather than inherent to rolling as a whole (see openSUSE tumbleweed for an example of what i'd consider a stable rolling distro). i'm open to discussion on this though, perhaps the wording could be improved to better reflect this?